megamek/src/megamek/server/GameManager.java

```diff
@@ -19481,6 +19479,11 @@ private void resolveHarJelRepairs() {
      * @param coordinates the coordinate location of the fire
      */
     private void doFlamingDamage(final Entity entity, final Coords coordinates) {
+        // TO:AR p.41,p.43
+        if (entity.tracksHeat() || entity.isDropShip()) {
```

Comment:
Don't all dropships track heat?

Reply:
DS technically are not heat-tracking units, since they never use the Heat Scale (even though their weapons usage is limited by heat sink capacity). They are explicitly called out as unaffected by fire in the TO:AR fire rules, however.

Reply:
tracksHeat() is only true for Meks, ASF and Small Craft. Dropships have an override to return false.